voctocore/lib/streamblanker.py

```diff
@@ -16,28 +16,30 @@ def __init__(self):
         self.log.info('Configuring StreamBlanker video %u Sources',
                       len(self.names))
 
+        self.volume = Config.getfloat('stream-blanker', 'volume', fallback=1.0)
```

Comment:
Everywhere else we have either a fallback, or an entry in the `default-config.ini`. Personally I actually prefer `fallback` over relying on the file. But I'm okay with any of:
* having both
* have it as a comment in the file
* remove the fallback and rely on the file.